x/blob/ante/blob_share_decorator_test.go

```diff
@@ -169,3 +170,53 @@ func TestBlobShareDecorator(t *testing.T) {
 func mockNext(ctx sdk.Context, _ sdk.Tx, _ bool) (sdk.Context, error) {
 	return ctx, nil
 }
+
+func TestBSDAppVersion(t *testing.T) {
+	type test struct {
+		appVersion uint64
+		isCheckTx  bool
+		expectErr  bool
+	}
+	tests := []test{
+		{appVersion: v1.Version, isCheckTx: false, expectErr: false},
```

Comment:
[question] can we make the antehandler behave the same regardless of `isCheckTx`? That way we can eliminate a whole class of these test cases.

If not, should we add a test case for `v1.Version` and `isCheckTx: true`?